### PR TITLE
user login

### PR DESCRIPTION
**Pull request by maximegavens**
template user login
with facebook login (don't work again)

### CODE REVIEW

File: frontend/src/App.vue [hunk 1]
```diff
@@ -25,5 +25,5 @@ export default {
 </script>
 
 <style lang="scss">
-@import "@/core/styles/index.scss";
+@import "core/styles/index.scss";
```
Comment (berthieresteban) [inline frontend/src/App.vue]: Why ? '@' is a shortcut for the root path of the project

File: frontend/src/LogIn/LogIn.vue [hunk 2]
```diff
@@ -0,0 +1,77 @@
+<template>
+  <v-layout
+    column
+    justify-center
+    align-center>
```
Comment (berthieresteban) [inline frontend/src/LogIn/LogIn.vue]: `v-layout` `column` and `v-flex` are from vuetify version 2.
We are using vuetify version 3, you need to use the grid system of that version.
https://vuetifyjs.com/en/components/grids#grid-system

File: frontend/package.json [hunk 3]
```diff
@@ -12,17 +12,22 @@
...
     "https": "^1.0.0",
     "lodash": "^4.17.15",
     "node-sass": "^4.13.0",
+    "regenerator-runtime": "^0.13.3",
     "roboto-fontface": "*",
     "sass-loader": "^8.0.0",
     "simple-oauth2": "^3.1.0",
     "vue": "^2.6.10",
     "vue-apexcharts": "^1.5.1",
+    "vue-facebook-login-component": "^1.4.0",
```
Comment (berthieresteban) [inline frontend/package.json]: You can remove this since it's not used

File: frontend/package.json [hunk 4]
```diff
@@ -12,17 +12,22 @@
...
+    "vue-facebook-signin-button": "^1.0.1",
```
Comment (berthieresteban) [inline frontend/package.json]: You can commit this on another branch to keep it for futures developments

File: frontend/src/LogIn/LogIn.vue [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+          </form>
+          <LogInFacebook/>
+        </v-card-text>
+      </v-card>
+    </v-flex>
+  </v-layout>
+</template>
+
+<script>
+import LogInFacebook from '../LogInFacebook/LogInFacebook.vue'
```
Comment (berthieresteban) [inline frontend/src/LogIn/LogIn.vue]: You can commit this on another branch to keep it for futures developments

File: frontend/src/LogIn/LogIn.vue [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+export default {
+  components: {
+    LogInFacebook
```
Comment (berthieresteban) [inline frontend/src/LogIn/LogIn.vue]: You can commit this on another branch to keep it for futures developments

File: frontend/src/LogInFacebook/LogInFacebook.vue [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+<style>
+  .fb-signin-button {
+    /* This is where you control how the button looks. Be creative! */
+    display: inline-block;
+    padding: 4px 8px;
+    border-radius: 3px;
+    background-color: #4267b2;
+    color: #fff;
+  }
+</style>
```
Comment (berthieresteban) [inline frontend/src/LogInFacebook/LogInFacebook.vue]: You can commit this on another branch to keep it for futures developments

File: frontend/src/main.js [hunk 8]
```diff
@@ -5,9 +5,10 @@ import store from './core/store'
 import vuetify from './core/plugins/vuetify'
 import 'roboto-fontface/css/roboto/roboto-fontface.css'
 import '@fortawesome/fontawesome-free/css/all.css'
+import FBSignInButton from 'vue-facebook-signin-button'
```
Comment (berthieresteban) [inline frontend/src/main.js]: You can commit this on another branch to keep it for futures developments

File: frontend/src/main.js [hunk 9]
```diff
@@ -5,9 +5,10 @@ import store from './core/store'
...
 import VueApexCharts from 'vue-apexcharts'
 Vue.use(VueApexCharts)
-
+Vue.use(FBSignInButton)
```
Comment (berthieresteban) [inline frontend/src/main.js]: You can commit this on another branch to keep it for futures developments

**Comment by berthieresteban** [comment]
Also, you probably forget to add your login road in the router file

File: frontend/src/App.vue (hunk 1 above)
Comment (maximegavens) [reply frontend/src/App.vue]: oh ok i got it

**Comment by maximegavens** [comment]
change commit.

**Pull request merged**